scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TFRecordIO.scala

```diff
@@ -88,6 +87,26 @@ object TFExampleIO {
   val WriteParam = TFRecordIO.WriteParam
 }
 
+final case class TFSequenceExampleIO(path: String) extends ScioIO[SequenceExample] {
```

Comment:
created a new `ScioIO` for the sequence example following how it was done with the exampleIO and the tfrecordIO, but both the `TFSequenceExampleIO ` and `TFExampleIO` can be deleted and the `fRecordSequenceExampleFile` and `fRecordExampleFile` methods can use the `TFRecordIO ` and just map and create the *Examples from bytes. Same applies to the write functions.